resources/js/Components/AuthWidget.vue

```diff
@@ -1,11 +1,13 @@
 <template>
-    <div class="auth-widget" v-if="user">
-        <a :href="`https://www.wikidata.org/wiki/User:${user.name}`">
-            <img src="images/user.svg" class="icon-user" /><span class="username">{{ user.name }}</span>
-        </a>
-        <a  href="/auth/logout">{{ $i18n('log-out') }}</a>
+    <div class="auth-widget_wrapper"> 
+        <div class="auth-widget" v-if="user">
+            <a :href="`https://www.wikidata.org/wiki/User:${user.name}`">
+                <img src="images/user.svg" class="icon-user" /><span class="username">{{ user.name }}</span>
+            </a>
+            <a  href="/auth/logout">{{ $i18n('log-out') }}</a>
+        </div>
+        <a class="auth-widget" v-else href="/auth/login">{{ $i18n('log-in') }}</a>
```

Comment:
Ah sorry, forgot to mention the class here needs to be removed, that's why we moved the css to the wrapper.